src/renderer/components/AircraftSection/index.tsx

```diff
@@ -87,6 +87,18 @@ const index: React.FC<TransferredProps> = (props: AircraftSectionProps) => {
     };
 
     const getTempDir = (): string => {
+        const communityDir = settings.get('mainSettings.msfsPackagePath') as string;
```

Comment:
Can we adapt this to #191 ?

Something like

```suggestion
        const tempDir = Directories.inCommunity(`flybywire_current_install_${(Math.random() * 1000).toFixed(0)}`);
```

Reply:
We'll just do it in #191.
Closing this